### PR TITLE
Implement L2NormToNorm transformation

### PR DESCRIPTION
**Pull request by evolosen**
Root cause analysis: Move L2 Normalize transformation to nGraph from MO

Ticket: CVS-72209

### CODE REVIEW

**Review by vgavrilo — APPROVED** [review]

File: src/tests/functional/inference_engine/transformations/common_optimizations/normalize_l2_fusion_test.cpp [hunk 1]
```diff
@@ -201,3 +201,141 @@ TEST_F(TransformationTestsF, NormalizeL2FusionWithAddIncorrectEpsValueShape) {
...
+
+        function = std::make_shared<ngraph::Function>(ngraph::NodeVector{mul}, ngraph::ParameterVector{input});
+
+        manager.register_pass<ngraph::pass::NormalizeL2Fusion>();
+    }
+
+    {
+        auto input =
+            std::make_shared<ngraph::opset4::Parameter>(ngraph::element::f16, ngraph::PartialShape::dynamic(3));
+        auto axes_const = ngraph::opset4::Constant::create(ngraph::element::i64, ngraph::Shape{2}, {0, 1});
```
Comment (GlebKazantaev) [inline src/tests/functional/inference_engine/transformations/common_optimizations/normalize_l2_fusion_test.cpp:228]: To check value in Constants during graph comparison please use:
```
comparator.enable(FunctionsComparator::CmpValues::CONST_VALUES);
```

File: src/tests/functional/inference_engine/transformations/common_optimizations/normalize_l2_fusion_test.cpp [hunk 2]
```diff
@@ -201,3 +201,141 @@ TEST_F(TransformationTestsF, NormalizeL2FusionWithAddIncorrectEpsValueShape) {
...
+        auto reduce_sum = std::make_shared<ngraph::opset4::ReduceSum>(pow, axes_const);
+        auto eps_const = ngraph::opset4::Constant::create(ngraph::element::f16, ngraph::Shape{}, {eps_value});
+        auto max = std::make_shared<ngraph::opset4::Maximum>(reduce_sum, eps_const);
+        auto sqrt_exp = ngraph::opset4::Constant::create(ngraph::element::f16, ngraph::Shape{}, {0.9f});
+        auto sqrt = std::make_shared<ngraph::opset4::Power>(max, sqrt_exp);
+        auto divide = std::make_shared<ngraph::opset4::Divide>(input, sqrt);
+
+        function_ref = std::make_shared<ngraph::Function>(ngraph::NodeVector{divide}, ngraph::ParameterVector{input});
+    }
```
Comment (GlebKazantaev) [inline src/tests/functional/inference_engine/transformations/common_optimizations/normalize_l2_fusion_test.cpp:340]: the same suggestion
```suggestion
```

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Comment by akladiev** [comment]
This PR was closed because it has been stalled for 2 week with no activity.

**Pull request closed without merging**